Gemfile

```diff
@@ -38,6 +38,7 @@ gem 'deacon', '~> 1.0'
 gem 'webpack-rails', '~> 0.9.8'
 gem 'mail', '~> 2.6'
 gem 'nokogiri', '< 1.7' if RUBY_VERSION.start_with? '2.0.'
+gem 'dynflow', '>= 0.8.20', '< 1.0.0'
```

Comment:
Set to `:path => '../dynflow` or similar when testing, the gem is not out yet for https://github.com/Dynflow/dynflow/pull/216/files